### PR TITLE
Scc 4162/vqa account settings

### PR DESCRIPTION
**Pull request by charmingduchess**
## Ticket:

- JIRA ticket [SCC-4162](https://newyorkpubliclibrary.atlassian.net/jira/software/c/projects/SCC/boards/126?assignee=61e836e95fcc37006878c15d&selectedIssue=SCC-4162)

## This PR does the following:

-    adds dashes to the phone number display

-    move update pin/password button

-    align pin password modal body with heading

-    Ensure the edit/save/cancel buttons move to the bottom of the tab on mobile view

## Accessibility concerns or updates

-

### Checklist:

- [ ] I updated the CHANGELOG with the appropriate information and JIRA ticket number (if applicable).
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: styles/components/MyAccount.module.scss
Comment (EdwinGuzman) [inline styles/components/MyAccount.module.scss:34]: Can you switch these over to use `min-width` instead? That'll follow the "mobile-first" approach. So in this case it'll be `flex-direction: column` be default and once the `min-width: 600px` value kicks in, it'll switch to `flex-direction: row`.

You can also use `var(--nypl-breakpoint-medium)` as the CSS var.

**Review by EdwinGuzman — COMMENTED** [review]
Looks good just one minor change but not a blocker.

Can you merge `main` in first before I approve? I want to see if this PR is affected by the updated styles in `main`.

**Review by EdwinGuzman — APPROVED** [review]
There's a minor responsive issue with the buttongroup at exactly 600px where it doesn't go into full width but I think that's a DS problem.

**Comment by charmingduchess** [comment]
> There's a minor responsive issue with the buttongroup at exactly 600px where it doesn't go into full width but I think that's a DS problem.

I had one of the min width set to 601 pixels. removed that and also rearranged the order of the styling to simplify.

**Pull request merged**
